help.html

```diff
@@ -170,24 +184,39 @@ <h3 id="Menu">Menu</h3>
                 </div>
             </td>
             <td>
-            Clicking on the "Snippets" button brings up a menu of Python
+            Clicking on the <strong>Snippets</strong> button brings up a menu of Python
             short-cuts (snippets). Code snippets are short blocks of code to
             re-use in your own programs. There are snippets for most common
             things you'll want to do with Python. Select the one you want and
             fill in the gaps in the code editor to make it do what you want.
             <strong class="hide-blocks">Snippets are disabled when blockly is enabled.</strong>
             </td>
         </tr>
+        <tr class="hide-options">
+            <td>
+                <div id="command-options" class="roundsymbol" title="Change the editor settings">
+                    <i class="fa fa-wrench"></i>
+                </div>
+            </td>
+            <td>
+            The <strong>Options</strong> menu allows users to change settings within the editor.
+            Clicking on the menu item will open a dropdown with the various options.
+
+            The <a href="#autocomplete">autocomplete</a> option can be switched on or off here.
+            When it is on you also have the option to complete code on pressing <kbd>Enter</kbd>
+            as well as <kbd>Tab</kbd>.   
+            </td>
+        </tr>
         <tr>
             <td>
                 <div id="command-help" class="roundsymbol" title="Find help and support">
                     <i class="fa fa-question"></i>
                  </div>
             </td>
             <td>
-            Clicking on the "Help" button gets you here. But you knew that
-            already, right..? :-) Notice that  editor is still available
-            but in a separate tab in  browser.
+            Clicking on the <strong>Help</strong> button opens a range of support options. From here
+            you can navigate to the MicroPython language reference documentation, this help page :),
+            the micro:bit support knowledgebase and the Python Editor issue tracker.
```

Comment:
Also need to mention the feedback link.

Reply:
done in https://github.com/bbcmicrobit/PythonEditor/pull/199/commits/715543570e673cd8361fce4aff56f62d3b59bc66